cuda:0
Traceback (most recent call last):
  File "/home/baebro/nipa_ws/nipaproj_ws/model/train.py", line 164, in <module>
    main()
  File "/home/baebro/nipa_ws/nipaproj_ws/model/train.py", line 67, in main
    model = skeletonLSTM(feature_dim, output_dim).to(device)
  File "/home/baebro/anaconda3/envs/mp/lib/python3.10/site-packages/torch/nn/modules/module.py", line 485, in __init__
    raise TypeError(
TypeError: skeletonLSTM.__init__() takes 1 positional argument but 3 were given
